phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/MainWindowView.java

```diff
@@ -74,16 +74,20 @@ protected void setSideBar(SideBar sideBar) {
         this.leftContent.setContent(sideBar);
     }
 
-    public void showRightView(Node nodeToShow) {
+    public void setCenter(Node nodeToShow) {
```

Comment:
What we should do here (in this method) too, is to call `closeDetailsView`. I think, that whenever this method is called the details view is automatically outdated.